src/fastoad/cmd/api.py

```diff
@@ -180,15 +183,22 @@ def list_variables(
         # For a terminal output, we limit width of NAME column
         variables_df["NAME"] = variables_df["NAME"].apply(lambda s: "\n".join(tw.wrap(s, 50)))
 
-    # In any case, let's break descriptions that are too long
-    variables_df["DESCRIPTION"] = variables_df["DESCRIPTION"].apply(
-        lambda s: "\n".join(tw.wrap(s, 100))
-    )
+    if variable_descriptions_format:
+        df_name_descriptions = variables_df[["NAME", "DESCRIPTION"]]
+        content = df_name_descriptions.to_csv(
+            sep="|", index=False, header=False, line_terminator="\n"
+        ).replace("|", " || ")
+        out_file.write(content)
+    else:
+        # In any other case, let's break descriptions that are too long
+        variables_df["DESCRIPTION"] = variables_df["DESCRIPTION"].apply(
+            lambda s: "\n".join(tw.wrap(s, 100))
+        )
 
-    out_file.write(
-        tabulate(variables_df, headers=variables_df.columns, showindex=False, tablefmt=tablefmt)
-    )
-    out_file.write("\n")
+        out_file.write(
+            tabulate(variables_df, headers=variables_df.columns, showindex=False, tablefmt=tablefmt)
+        )
+        out_file.write("\n")
```

Comment:
I think these two very different ways of working would justify to create 2 private methods.

Reply:
Thanks. I proposed a solution in 44b3483.